src/components/placement/PlacementHero.astro

```diff
@@ -0,0 +1,46 @@
+---
+import CountUpComponent from "../Hero/CountUpComponent";
+---
+<div class="flex lg:flex-row flex-col items-center gap-12 w-full mt-12 mb-12">
+  <div class="w-full sm:order-2 lg:order-1 order-1">
+    <h1 class="lg:text-4xl text-3xl mb-4 font-title">About Placement</h1>
+    <p class="mb-8 lg:text-base text-md text-slate-500">
+      The Training and Placement department serves as a bridge between the
+      academic environment and the industry by establishing connections with
+      various companies and organizations. The ultimate aim of the T&P
+      department is to ensure that students are well-prepared for the job market
+      and have access to the resources they need to achieve their career goals.
+    </p>
+    <button
+      class="text-slate-500 hover:bg-slate-100 active:bg-slate-50 transition-all border border-gray-300 rounded-xl px-4 py-2"
+      >Visit our Guidelines</button
+    >
+  </div>
+  <div class="w-full sm:order-1 lg:order-2 order-2">
+    <div class="mb-6 flex flex-col justify-center gap-6 rounded-xl">
+      <img
+        src="/Placement/placement.png"
+        class="w-full aspect-auto rounded-xl"
+        alt="image-of-a-female-student-pointing-towards-a-laptop-screen"
+      />
+      <div class="flex justify-between sm:flex-row flex-col items-center gap-3">
```

Comment:
Make the box sizes big so that it covers the image on lg screen check figma how the space is occupied.
![image](https://user-images.githubusercontent.com/79577834/232308995-c5c0d855-fa5a-444d-9b51-8c6521879054.png)
It should appear like this

Reply:
1. I purposely kept it for lg, as image up was looking good to me, but ok ill remove it.

Reply:
just remove from  the line I mentioned because when you were adding this the image was still at top after a certain range that's why ..